core/src/test/java/org/elasticsearch/cluster/NoMasterNodeIT.java

```diff
@@ -121,23 +121,21 @@ public void run() {
                 ClusterBlockException.class, RestStatus.SERVICE_UNAVAILABLE
         );
 
-        checkWriteAction(
-                false, timeout,
+        checkWriteAction(timeout,
                 client().prepareUpdate("test", "type1", "1")
                         .setScript(new Script(
                             ScriptType.INLINE, Script.DEFAULT_SCRIPT_LANG, "test script", Collections.emptyMap())).setTimeout(timeout));
 
-        checkWriteAction(
-                autoCreateIndex, timeout,
+        checkWriteAction(timeout,
                 client().prepareUpdate("no_index", "type1", "1")
                         .setScript(new Script(
                             ScriptType.INLINE, Script.DEFAULT_SCRIPT_LANG, "test script", Collections.emptyMap())).setTimeout(timeout));
 
 
-        checkWriteAction(false, timeout,
+        checkWriteAction(timeout,
                 client().prepareIndex("test", "type1", "1").setSource(XContentFactory.jsonBuilder().startObject().endObject()).setTimeout(timeout));
 
-        checkWriteAction(autoCreateIndex, timeout,
```

Comment:
why can we remove autoCreateIndex? I mean it can still be created? what am I missing?